### ISSUE TITLE
Larger thumbnails

### ISSUE THREAD

**Issue by adarshv90** [lhc70000/iina#885]
Make thumbnails larger. Better yet, give the option for custom thumbnail sizes

**System and IINA version:**

- macOS 10.13
- IINA 0.0.12

**Comment by JohnClay** [comment]
This would be very nice.

**Comment by kgoldmann** [comment]
Agreed. For comparison:

Netflix
<img width="1535" alt="screen shot 2018-02-22 at 03 28 25" src="https://user-images.githubusercontent.com/8964638/36516963-a06704c2-1780-11e8-9d23-62c29675c445.png">

IINA
<img width="1439" alt="screen shot 2018-02-22 at 03 29 00" src="https://user-images.githubusercontent.com/8964638/36516968-a4fbedae-1780-11e8-91d2-55a9a62daa09.png">

**Comment by firstel** [comment]
I would just like thumbnails to work over smb.  this command works - but only once.  

defaults write com.colliderli.iina enableThumbnailForRemoteFiles -bool YES

**Comment by inflation** [comment]
@firstel Please open another issue or reply in the relevant issue.

**Comment by bozorgmehr96** [comment]
Do we get larger thumbnails ever?

**Comment by joeynotjoe** [comment]
Adding my vote to this, would love to be able to choose what size we display the thumbnail at, as well as what interval they are generated.

**Comment by AJeezy9** [comment]
Please please please please please please please please 😄

**Comment by triemilalim** [comment]
I almost make a new feedback, turns out someone already asked for this 3 years ago. Please give us the option to make the thumbnail seeker bigger, or even custom thumbnail sizes. Thanks guys!!

**Comment by anastasiuspernat** [comment]
I made a pull request for this one! Hopefully @lhc70000 or other IINA demiurge else will have time to review and integrate it if it works out that way. Thanks!

**Comment by TitanicKiwi** [comment]
I'm adding myself to this request. Please add the option to set our own size for the thumbnail preview.

**Comment by rogery555** [comment]
unbelievable this is still open, thumbnails are barely usable they are so small

**Comment by Careless** [comment]
come on alreaddddddddddddy

**Comment by FaysalAriss** [comment]
Seems to be on the way, but if you're in a hurry download the latest nightly (currently 7bc60e6) and in a terminal run: 
defaults write com.colliderli.iina thumbnailWidth -int 600
Replace 600 with any width you want.

**Comment by uiryuu** [comment]
If you are using the nightly builds, you can try out our new settings window by running this command:
```
defaults write com.colliderli.iina enableNewSettings -bool Yes
```
The thumbnail width adjustment in available in the new settings window.

**Comment by Careless** [comment]
> If you are using the nightly builds, you can try out our new settings window by running this command:
> 
> ```
> defaults write com.colliderli.iina enableNewSettings -bool Yes
> ```
> 
> The thumbnail width adjustment in available in the new settings window.

this works. the adjustment is in the interface settings tab. 
i like the switch for the mounted drives as well.

360px width is a nice size for a 1440p monitor. 
Youtube's preview size is 240 * 420px if anyone wants to match it.

thank you for adding this!
